eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/OnChainFeeConf.scala

```diff
@@ -79,11 +68,10 @@ case class OnChainFeeConf(feeTargets: FeeTargets, feeEstimator: FeeEstimator, sp
    * @param channelType         channel type
    * @param currentFeerates_opt if provided, will be used to compute the most up-to-date network fee, otherwise we rely on the fee estimator
    */
-  def getCommitmentFeerate(remoteNodeId: PublicKey, channelType: SupportedChannelType, channelCapacity: Satoshi, currentFeerates_opt: Option[CurrentFeerates]): FeeratePerKw = {
-    val (networkFeerate, networkMinFee) = currentFeerates_opt match {
-      case Some(currentFeerates) => (currentFeerates.feeratesPerKw.feePerBlock(feeTargets.commitmentBlockTarget), currentFeerates.feeratesPerKw.mempoolMinFee)
-      case None => (feeEstimator.getFeeratePerKw(feeTargets.commitmentBlockTarget), feeEstimator.getMempoolMinFeeratePerKw())
-    }
+  def getCommitmentFeerate(remoteNodeId: PublicKey, channelType: SupportedChannelType, channelCapacity: Satoshi): FeeratePerKw = {
```

Comment:
I removed the `currentFeerates_opt: Option[CurrentFeerates]`. The reason for this argument was that when the channel received a `CurrentFeerateEvent` and calls `getCommitmentFeerate`, it has to be sure that the latest feerate will be used.

I think it's okay to make the assumption that the atomic reference is updated before  `CurrentFeerate` is emitted, which is already the case in our implementation.

A drawback is that there will be a herd effect every 10min, where all channels will attempt to read the atomic reference at the same time. Is it a big deal? I'm not sure what the locking contraints are. We can work around that by using the same trick as `ProcessCurrentBlockHeight`, which was introduced for that purpose.